scalatest/src/main/scala/org/scalatest/Suite.scala

```diff
@@ -2088,7 +2088,7 @@ used for test events like succeeded/failed, etc.
 
   // SKIP-SCALATESTJS,NATIVE-START
   def getMethodForTestName(theSuite: org.scalatest.Suite, testName: String): Method = {
-    val candidateMethods = theSuite.getClass.getMethods.filter(_.getName == Suite.simpleNameForTest(testName))
+    val candidateMethods = theSuite.getClass.getMethods.map(_.nn).filter(_.getName == Suite.simpleNameForTest(testName))
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.